packages/gatsby-plugin-typescript/README.md

```diff
@@ -9,6 +9,7 @@ Provides drop-in support for TypeScript and TSX.
 ## How to use
 
 1.  Include the plugin in your `gatsby-config.js` file.
+1.  Install related type definition based on your installed packages.
```

Comment:
When we introduce a change like this, we need to weigh whether it's improving comprehension or just introducing potential confusion.

In this case, I'd wager:

1. Someone using this plugin (and more broadly speaking, TypeScript) would generally _already_ know this (?), or
2. If they don't--this doesn't seem like it'd necessarily be helpful to that person?
    - In other words, what action would this cause a reader to take to fix the warning?

So, I'd lean towards just keeping this the way it is before this change. What do you think?

Reply:
Well, I'm new in TypeScript world and maybe you right. But assume people already know this I think a bit harsh(?)

And maybe I'm biased from how create-react-app explicitly said to install type definition (not assume people already know Typescript or not)

https://facebook.github.io/create-react-app/docs/adding-typescript#docsNav

Reply:
> If they don't know, this can help them to fix the error above.

I'm with Dustin here. The current sentence wouldn't help them and rather cause confusion. It's also not necessary in the context of setting up this plugin (which this README is about).

How about a paragraph below the steps like:

_**Please note:** If packages don't ship with TypeScript definitions you'll need to manually install those type definitions, e.g. for React. A typical Gatsby project would need: `npm install --save-dev @types/react @types/react-dom @types/node`._

---

Because `react-helmet` is too specific IMO. Not everyone is using that.

Reply:
Sure, that's better!

Sorry, it's my bad can't arrange the sentence properly. My main concern is mention that to everyone whose use this plugin. So no problem for the notes above.

And before, I think `react-helmet` is used in all gatsby template so I'm include that there. Thanks for clarification.

Reply:
Cool :) You can go ahead @iwgx and make the change!

Reply:
@LekoArts 👍 that sounds awesome. As mentioned above, we aren't as opinionated as create-react-app so it makes sense to be as general as we can.